blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/section-nav.jsx

```diff
@@ -88,7 +88,7 @@ const SubSectionAnchor = ({ item, isOpen }) => (
   <div className={`subsection-anchor ${isOpen ? 'open' : ''}`} onClick={onClickSubsection}>
     <SectionAnchor item={item} />
     <span className="submenu-caret">
-      <ChevronRight height={24} width={24} />
+      <ChevronRight height={20} width={20} />
```

Comment:
This change should be applied to `header-nav-block` as well

Reply:
`header-nav-block` was not updated with the new behavior and style of `header-nav-chain-block`. 
`header-nav-block` will be deprecated.